frontend/src/actions/index.js

```diff
@@ -0,0 +1,11 @@
+// index.js file that creates an action.
+
+import { CHANGE_MAP_STATE } from "../constants/action-types";
+
+export function changeMapState(payload) {
+    console.log("hello from actions! ");
+    console.log(payload);
+  return {
+    type: CHANGE_MAP_STATE, payload
+  };
+}
```

Comment:
we usually add new line at the end of the file.  you should also add one to all pages.